static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -150,6 +150,23 @@ define([
         });
     }
 
+    function initHeroic(){
+        if($('.vjs-tech') && $('.labour-liverpool-headline') && $('.vjs-control-bar')){
```

Comment:
just to avoid doing `$` twice you can move to after the `var`s and do `if (player && headline && controls)`